ethereum/core/src/main/java/org/hyperledger/besu/ethereum/trie/diffbased/verkle/storage/VerkleWorldStateKeyValueStorage.java

```diff
@@ -164,6 +207,19 @@ public Updater putAccountInfoState(final Hash accountHash, final Bytes accountVa
       return this;
     }
 
+    public synchronized Updater putStorageValueBySlotHash(
```

Comment:
This Updater has a bunch of methods `synchronized` but then others are not. Does it make sense to have `synchronized` at all? If so others are also writing to the same object as these

Reply:
I think it's just a copy/paste from Bonsai code, looking at the code I also don't see a need for a synchronized for the moment